src/deepsparse/image_classification/pipelines.py

```diff
@@ -148,55 +148,10 @@ def process_inputs(self, inputs: ImageClassificationInput) -> List[numpy.ndarray
         if isinstance(inputs.images, numpy.ndarray):
```

Comment:
So if there's a numpy array, the images don't get preprocessed?

Reply:
Yep that's the existing behavior, so want to keep that the same!

Reply:
Ha, a conversation about this is probably outside of the scope of this PR. Good catch parallelizing the preprocessing.

Reply:
Agree 😄 I'll add an issue to the backlog for it